single-node/Vagrantfile

```diff
@@ -21,9 +21,9 @@ Vagrant.configure("2") do |config|
   # always use Vagrant's insecure key
   config.ssh.insert_key = false
 
-  config.vm.box = "coreos-alpha"
+  config.vm.box = "coreos-beta"
```

Comment:
I'm assuming this change was due to: https://github.com/coreos/bugs/issues/1246 - but now that it is resolved, can we switch this back to alpha?

I'm not necessarily opposed to changing this, just checking motivation.